src/main/java/com/nerodesk/om/mock/MkFriends.java

```diff
@@ -71,6 +76,15 @@ public MkFriends(final File file, final String urn, final String name) {
         this.dir = file;
         this.user = urn;
         this.label = name;
+        this.friends = Paths.get(
+            this.dir.getAbsolutePath(),
+            this.user,
```

Comment:
@pinaf to avoid illegal symbols you could use something like
```
this.name.replaceAll("[^a-z0-9]", File.separator)
```

This is how it was fixed at `MkUser.docs` for example.